applicationinsights-react-js/src/ReactPlugin.ts

```diff
@@ -33,12 +36,19 @@ export default class ReactPlugin extends BaseTelemetryPlugin {
         let _unlisten: any;
         let _pageViewTimer: any;
         let _pageViewTracked:boolean;
+        let properties: PropertiesPlugin;
+
 
         dynamicProto(ReactPlugin, this, (_self, _base) => {
             _initDefaults();
 
             _self.initialize = (config: IConfiguration & IConfig, core: IAppInsightsCore, extensions: IPlugin[], pluginChain?:ITelemetryPluginChain) => {
                 super.initialize(config, core, extensions, pluginChain);
+                
+                properties = new PropertiesPlugin();
```

Comment:
This should "lookup" the core instance for the current "PropertiesPlugin" instance, just creating and returning a new instance here is not going to affect the pipeline processing.

```ts
let thePlugin = core.getPlugin<PropertiesPlugin>("<the properties plugin name>");
if (thePlugin) {
return thePlugin.plugin;
}
```

Reply:
We can use the `PropertiesPluginIdentifier` (from the common package) as the string above that I called "<the p[roperties plugin name">